packages/material/scss/_variables.scss

```diff
@@ -424,8 +452,84 @@ $icon-size: 24px;
 $icon-size-lg: 32px;
 $icon-spacing: 4px !default;
 
+// Disabled mixin variables
+$disabled-filter: grayscale(.1) !default;
+$disabled-opacity: .6 !default;
+
 // Pager
 $pager-padding-x: 8px !default;
 $pager-padding-y: 10px !default;
 
+
+// Card
+$card-padding-x: 16px !default;
+$card-padding-y: 16px !default;
+$card-border-width: 0 !default;
+$card-border-radius: 2px !default;
+$card-box-shadow: mat-elevation-value(2) !default;
+
+$card-deck-gap: 16px !default;
+
+$card-bg: $panel-bg !default;
+$card-text: $panel-text !default;
+$card-border: $panel-border !default;
+
+$card-header-padding-x: $card-padding-x !default;
+$card-header-padding-y: $card-padding-y !default;
+$card-header-border-width: 0 !default;
+$card-header-bg: false !default;
+$card-header-text: false !default;
+$card-header-border: false !default;
+
+$card-body-padding-x: $card-padding-x !default;
+$card-body-padding-y: $card-padding-y !default;
+
+$card-actions-padding-x: $card-padding-x !default;
+$card-actions-padding-y: ($card-padding-y - $button-padding-y) !default;
+$card-actions-border-width: 0 !default;
+
+
+// Config maps
+$layout-config: () !default;
+// sass-lint:disable indentation
+$layout-config: map-merge((
+    display-1: layout-map(),
+    display-2: layout-map(),
+    display-3: layout-map(),
+    display-4: layout-map(),
+    h1: layout-map(0 0 16px),
+    h2: layout-map(0 0 16px),
+    h3: layout-map(0 0 16px),
+    h4: layout-map(0 0 16px),
+    h5: layout-map(0 0 12px),
+    h6: layout-map(0 0 12px),
+    p: layout-map(0 0 12px)
+), $layout-config);
+$layout-config: map-merge((
+    card-title: map-get($layout-config, h1),
+    card-subtitle: map-get($layout-config, p)
+), $layout-config);
+// sass-lint:enable indentation
+
+$typography-config: () !default;
+// sass-lint:disable indentation
+$typography-config: map-merge((
+    display-1: font-map($display1-font-size, 112px, $font-weight-light, $font-family, -.05em),
+    display-2: font-map($display2-font-size, 56px, $font-weight-normal, $font-family, -.02em),
+    display-3: font-map($display3-font-size, 48px, $font-weight-normal, $font-family, -.005em),
+    display-4: font-map($display4-font-size, 40px, $font-weight-normal, $font-family),
+    h1: font-map($h1-font-size, 32px, $font-weight-normal, $font-family),
+    h2: font-map($h2-font-size, 32px, $font-weight-bold, $font-family),
+    h3: font-map($h3-font-size, 28px, $font-weight-normal, $font-family),
+    h4: font-map($h4-font-size, 24px, $font-weight-normal, $font-family),
+    h5: font-map($h5-font-size, $line-height, $font-weight-normal, $font-family),
+    h6: font-map($h6-font-size, $line-height, $font-weight-normal, $font-family),
+    body: font-map($font-size, $line-height, $font-weight-normal, $font-family)
+), $typography-config);
+$typography-config: map-merge((
```

Comment:
As for the typography configuration, would it work if the styles are applied to the selectors directly in a `_theme.scss` file and using `@extends` to inherit the typography styles in the card? For example:

```scss
// instead of the custom DSL...
$typography-config: map-merge((
    // ...
    h1: font-map($h1-font-size, 32px, $font-weight-normal, $font-family),
    // ...
    body: font-map($font-size, $line-height, $font-weight-normal, $font-family)
), $typography-config);
$typography-config: map-merge((
    card-title: map-get($typography-config, h1),
    card-subtitle: map-get($typography-config, body)
), $typography-config);

// ... we can apply styles directly to the selectors
// _theme.scss
.k-h1 {
    font-size: $h1-font-size;
    line-height: 32px;
    font-weight: $font-weight-normal;
    font-family: $font-family;
}

.k-card-title {
    @extend .k-h1;
}
```